homeassistant/components/local_calendar/config_flow.py

```diff
@@ -35,6 +61,31 @@ async def async_step_user(
         key = slugify(user_input[CONF_CALENDAR_NAME])
         self._async_abort_entries_match({CONF_STORAGE_KEY: key})
         user_input[CONF_STORAGE_KEY] = key
+        if url := user_input.get(CONF_CALENDAR_URL):
```

Comment:
Increase test coverage for this file.

Reply:
Sure, tests and documentation are a must for the merge. I was coding this from my phone, on my live home assistant instance and I did not have access to tools to run the testsuite.

I will add the tests and a documentation pull request as well